noir_stdlib/src/string.nr

```diff
@@ -0,0 +1,11 @@
+use crate::collections::vec::Vec;
+impl<N> str<N> {
+    /// return a byte slice of the str content
+    #[builtin(str_as_bytes)]
+    fn as_bytes(_self: Self) -> [u8] { }
+
+    /// return a byte vector of the str content
+    fn into_bytes(self: Self) -> Vec<u8> {
```

Comment:
Can we rename this to `as_bytes_vec` or similar? `into` from rust implies we're moving the string but that is not the case here.